src/prompts/init.ts

```diff
@@ -0,0 +1,27 @@
+import { type InferSchema, type PromptMetadata } from "xmcp"
+import { z } from "zod"
+import { readResourceFile } from "../_internal/resource-path"
+
+export const schema = {
+  project_path: z
+    .string()
+    .optional()
+    .describe("Path to the Next.js project (defaults to current directory)"),
+}
+
+export const metadata: PromptMetadata = {
+  name: "init",
+  title: "initialize-next-js-dev-tools-context",
+  description:
+    "Sets up the context for working with Next.js applications using the Next.js DevTools MCP server. Explains available tools and establishes the requirement to use Next.js official documentation for all Next.js-related queries.",
+  role: "user",
+}
+
+export default function getInitPrompt(args: InferSchema<typeof schema>): string {
+  const projectPath = args.project_path || process.cwd()
```

Comment:
prefer not give fallback which could cause confusion in the future, like which case is this for 
```suggestion
  const projectPath = args.project_path
```